### PR TITLE
feat(docx): update docs inline table styling [JOB-110350]

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

The markdown tables in docs did not look great

image

### Changed

- Styling for markdown-generated tables in new docs site
image

## Testing

Look at 
- Button (Content guidelines)
- Animation, Spacing, Radius design docs

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/site/src/main.css [hunk 1]
```diff
@@ -40,51 +40,55 @@ a:visited {
   color: var(--color-text--secondary);
 }
 
-custom-elements {
+custom-elements,
+section[role="tabpanel"] {
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:44]: @scotttjob I tacked the selector for these changes onto your existing `custom-elements` selector, noticed that the `custom-elements` wrapper isn't present on an individual component page the way it is in a design token's docs

File: packages/site/src/main.css [hunk 2]
```diff
@@ -40,51 +40,55 @@ a:visited {
...
     -webkit-overflow-scrolling: touch;
-    margin: 16px 0;
-    margin-bottom: 16px;
+    margin: var(--space-large) auto;
     font-size: 14px;
     line-height: 24px;
     padding: 0;
     border-collapse: collapse;
+    box-shadow: var(--shadow-low);
+    border-radius: var(--radius-base);
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:58]: When a table has `border-collapse` applied, `border-radius` stops working, so the box-shadow gives us the visual outline without using a border! (You can also see this in the Atlantis `<Table>` component, thanks @eddysims for showing me this all those years ago)

File: packages/site/src/main.css [hunk 3]
```diff
@@ -40,51 +40,55 @@ a:visited {
...
+    overflow: hidden;
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:59]: Crops the background-color of the `thead`

File: packages/site/src/main.css [hunk 4]
```diff
@@ -40,51 +40,53 @@ a:visited {
...
   }
 
   & thead {
-
-    font-size: 16px;
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:61]: See redundant declaration 3 lines below

File: packages/site/src/main.css [hunk 5]
```diff
@@ -40,51 +40,55 @@ a:visited {
...
 
   & thead {
-
     font-size: 16px;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-size: 14px;
     line-height: 24px;
+    text-align: start;
+    background-color: var(--color-surface--background);
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:67]: image

File: packages/site/src/main.css [hunk 6]
```diff
@@ -40,51 +40,53 @@ a:visited {
...
-    margin: 0;
+    min-width: 66%;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
     -webkit-overflow-scrolling: touch;
-    margin: 16px 0;
-    margin-bottom: 16px;
-    font-size: 14px;
+    margin: var(--space-large) auto;
```
Comment (chris-at-jobber) [inline packages/site/src/main.css:51]: They sit in the centre now - open to keeping them left-aligned though

**Review by taylorvnoj — APPROVED** [review]
LGTM! Great update!

**Pull request merged**